readme.md

```diff
@@ -139,8 +166,15 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
+function Instructor(name){
+  this.name = name;
+  this.receivePresent = function(gift){
```

Comment:
:+1: Definitely works, but just a note that It may be more appropriate to write this function as an instance method